src/editors/containers/VideoUploadEditor/index.jsx

```diff
@@ -95,17 +88,11 @@ const VideoUploadEditor = (
   const {
     loading,
     setLoading,
-    errorMessage,
-    setErrorMessage,
   } = hooks.uploadEditor();
   const handleCancel = editorHooks.handleCancel({ onClose });
```

Comment:
It seems if you provide onClose as an argument to this function it returns that function, so it is doing nothing here. This function take onClose and returns onClose. So this call can be skipped.

Reply:
It is going with the editor component which this component is wrapped in, FWIW it's telling the parent component about it. I am keeping this for now.

Reply:
I'm not sure I understand what you mean? `handleCancel` is just a regular function. I think you can replace this with
`  const handleCancel = onClose;` and it will work the same. 

It's not part of this PR, so it's OK to leave it for now.

Reply:
This is required to get the close button working on the editor since this is passed as a parameter to a higher-order function.